public/factsheets/sample.json

```diff
@@ -0,0 +1,20 @@
+{
+  "metadata": {
+    "includedYears": [2014, 2015, 2016]
+  },
+  "data": [
```

Comment:
Note: the API is similar but not exactly the same. Please have a look at the swagger UI for more details.